agate/pattern-single-panel-redux/src/views/MainPanel.js

```diff
@@ -32,10 +33,17 @@ const MainPanel = kind({
 	name: 'MainPanel',
 
 	render: (props) => (
-		<Panel {...props}>
-			<Header casing="preserve" title="Profile Photo Redux" titleBelow="Choose your profile picture" type="compact" />
-			<Column align="center">
-				<Cell>
+		<Panel {...props} style={{paddingTop: '0px'}}>
```

Comment:
I don;'t think it needed.
is there any reason to override the padding?
```suggestion
		<Panel {...props}>
```

Reply:
I added paddingTop to make the save button not look cropped, but now I made the save button smaller and removed paddingTop property.